core/repository/block.ts

```diff
@@ -99,12 +102,13 @@ class BlockRepo {
             if (!result) {
                 return new Response({ errors: ['No blocks found']});
             }
-            result.forEach((row) => {
-                blocks.push(this.dbRead(row));
+            result.forEach(async (row) => {
```

Comment:
forEach works incorrectly with async/await and you need to use for...of

https://codeburst.io/javascript-async-await-with-foreach-b6ba62bbf404